app/src/main/java/com/henagle/www/kanban/model/CardSerializer.java

```diff
@@ -1,8 +1,10 @@
-package com.henagle.www.kanban;
+package com.henagle.www.kanban.model;
```

Comment:
Nit: This isn't really a model it's more like a utility class. It doesn't really follow any OO principles so I'm trying to figure out what a reasonable solution is for it.

Reply:
Would it make sense to make a utility package?

Reply:
This is something we'll keep in mind moving forward